src/devices/inovelli.ts

```diff
@@ -478,6 +478,8 @@ const COMMON_ATTRIBUTES: {[s: string]: Attribute} = {
         max: 127,
         readOnly: true,
         description: 'The temperature measured by the temperature sensor inside the chip, in degrees Celsius',
+        category: 'diagnostic',
+        unit: '°C',
```

Comment:
@Koenkk - I just realized that on all of the Inovelli devices, the internalTemperature parameter doesn't show up as the `device_class: temperature` when MQTT sends it to Home Assistant. I thought adding the unit and category would fix that, but it doesn't seem to be doing so. Is there something else that needs to be set?

Reply:
For that it needs to be added here; https://github.com/Koenkk/zigbee2mqtt/blob/bc6eccf6152c4cd91e75309f7ea0b8ad4e8835d4/lib/extension/homeassistant.ts#L294

Reply:
Cool - since it's a separate repo, we can handle it separately after this is merged if that's okay with you.